src/topology/algebra/continuous_affine_map.lean

```diff
@@ -0,0 +1,147 @@
+/-
+Copyright (c) 2021 Oliver Nash. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Oliver Nash
+-/
+import linear_algebra.affine_space.affine_map
+import topology.metric_space.basic
+import topology.continuous_function.basic
+
+/-!
+# Continuous affine maps.
+
+This file defines a type of bundled continuous affine maps.
```

Comment:
Could you add an explanation of the point we were discussing, why `continuous_affine_map.cont_linear` has to be deferred?

Reply:
Good idea; done.